content/guides/digging-deeper/rate-limiting.md

```diff
@@ -88,6 +88,21 @@ The Database store relies on the `@adonisjs/lucid` package. Therefore, make sure
 
 The database connection details are defined inside the `config/database.ts` file. In addition, you should mention the connection's name inside the limiter's config file.
 
+```ts
+export default limiterConfig({
+  default: 'db',
+  stores: {
+    db: {
+      client: 'db',
+      dbName: Env.get('PG_DB_NAME'),
```

Comment:
How about not using the env variable for a specific database dialect? I will instead prefer a raw string.

```ts
dbName: 'database_name'
```

Reply:
Okk, good to now. I will change this